src/main/java/com/vinylteam/vinyl/dao/jdbc/mapper/UserRowMapper.java

```diff
@@ -1,4 +1,34 @@
 package com.vinylteam.vinyl.dao.jdbc.mapper;
 
+import com.vinylteam.vinyl.entity.Role;
+import com.vinylteam.vinyl.entity.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Base64;
+
 public class UserRowMapper {
+
+    private static final Logger logger = LoggerFactory.getLogger(UserRowMapper.class);
```

Comment:
And here it is static? We should avoid static when we can do it.

Reply:
Ok